charts/envelope-game/templates/deployment.yaml

```diff
@@ -54,11 +50,12 @@ spec:
           livenessProbe:
             httpGet:
               path: /
-              port: http
+              port: 3000
```

Comment:
we probably don't need to change this.  it's nice to reference a port by its name so we only have to change it in a single place.

Reply:
reverting these, thanks!